lib/Doctrine/DBAL/Platforms/Keywords/MySQL57Keywords.php

```diff
@@ -241,6 +242,7 @@ protected function getKeywords()
             'SSL',
             'STARTING',
             'STRAIGHT_JOIN',
+            'STORED',
```

Comment:
This should be placed after `'STARTING'` and before `'STRAIGHT_JOIN'` in order to keep the current sorting.